### ISSUE TITLE
Wrong subfolder delimiter on Windows

### ISSUE THREAD

**Issue by kimbar** [gimenete/github-linker#1]
On Windows extension produces windows-flavour paths for files in subfolders:
```
https://github.com/user/project/blob/0123456789abcdef/folder\subfolder\file.js#L19-L21
```
should be:
```
https://github.com/user/project/blob/0123456789abcdef/folder/subfolder/file.js#L19-L21
```

**Comment by gimenete** [comment]
Should be fixed in the latest version (0.2.3). Please let me know if you find any issues.

Thanks!

**Comment by kimbar** [comment]
Great! Checked, works fine.
